src/systemd/sd-bus.h

```diff
@@ -209,10 +209,10 @@ int sd_bus_wait(sd_bus *bus, uint64_t timeout_usec);
 int sd_bus_flush(sd_bus *bus);
 int sd_bus_enqueue_for_read(sd_bus *bus, sd_bus_message *m);
 
-sd_bus_slot* sd_bus_get_current_slot(sd_bus *bus);
-sd_bus_message* sd_bus_get_current_message(sd_bus *bus);
+sd_bus_slot *sd_bus_get_current_slot(sd_bus *bus);
+sd_bus_message *sd_bus_get_current_message(sd_bus *bus);
```

Comment:
... but this one, I think it makes things harder to read. I.e. above, `name` is a pointer, so attaching the modifier to it is appropriate, but here, sd_bus_get_current_slot is also a pointer (to a function, already without the modifier), and it returns something (a different pointer). By attaching the modifier to the return type, we make it clearer that the modifier applies to that, and not the function.

I know that the compiler doesn't care, so this is not something that has a right and a wrong answer, but please drop the parts that move the modifier to the function symbol.

Reply:
Alright, I did this without thinking too much about it except that stuff didn't seem consistent but you make a good argument.

Reply:
What about cases like the following?

```
const char *sd_bus_message_get_signature(sd_bus_message *m, int complete);
const char *sd_bus_message_get_path(sd_bus_message *m);
const char *sd_bus_message_get_interface(sd_bus_message *m);
const char *sd_bus_message_get_member(sd_bus_message *m);
const char *sd_bus_message_get_destination(sd_bus_message *m);
const char *sd_bus_message_get_sender(sd_bus_message *m);
```

Do we use `const char* sd_bus` here? I feel like `const char * sd_bus` could work as well. Maybe two spaces could work for all return types since the return type is formatted as a variable declaration without a name, e.g. `const char * sd_bus` makes it clear we're returning an unnamed `const char *` from the function. I'm fine with both but I thought I'd mention the possibility. I don't suddenly want to diverge from any established styles so if `const char*` would normally be used we can keep that.

Reply:
My preference would be for `const char*`. `*` surrounded by spaces looks too much like multiplication, and also we almost never use that style.